src/docs/sphinx/pythonTools/mesh_doctor.rst

```diff
@@ -122,3 +122,60 @@ It will also verify that the ``VTK_POLYHEDRON`` cells can effectively get conver
 
 .. command-output:: python mesh_doctor.py supported_elements --help
    :cwd: ../../../coreComponents/python/modules/geosx_mesh_doctor
+
+``Using mesh_doctor in paraview``
+""""""""""""""""""""""""""""""""""
+To use ``mesh_doctor`` in Paraview as a python programmable filter, a python package install is required first in Paraview python resolved
+path. Paraview is storing its python ressources under its *lib/pythonX.X* depending on the paraview version, *e.g* Paraview 5.11 is working
+with python 3.9. As a results the following command will install ``mesh_doctor`` package into Paraview resolved path.
+
+.. command-output:: python3 -m pip install --index-url https://test.pypi.org/simple/ --no-deps --upgrade --target /path/to/Paraview/lib/python3.9/ mesh_doctor
+
+.. note::
+    ``pip`` is installing the ``mesh_doctor`` package from the test.pypi repo, which is intended to test package deployment.
+    Once stabilized and ``mesh_doctor`` uploaded onto the main package repo, this should be dropped out.
```

Comment:
<img width="325" alt="image" src="https://github.com/GEOS-DEV/GEOS/assets/49004943/51eda42b-a2f2-4b4e-933f-d0256340a0b1">
😱 🤣

Reply:
Hacky I said :stuck_out_tongue_closed_eyes:

Reply:
is installing it like this working for you? When I tried it seemed to me that there were some issue in the setup.py that did not allow for this to work. Basically I could not find the modules when trying to import them.

Reply:
I got this output 
```
Looking in indexes: https://test.pypi.org/simple/
Collecting mesh_doctor
  Using cached https://test-files.pythonhosted.org/packages/97/cc/1ece6916f6e87f95cff4b887f2e6ce50488b146a2e025a959c3665a12d6e/mesh_doctor-0.0.1-py3-none-any.whl (42 kB)
Installing collected packages: mesh_doctor
Successfully installed mesh_doctor-0.0.1
```
and I can checked afterwards that `checks` is in `/path/to/Paraview/lib/PythonX.X' but this need refactor/cleanup obv.

Reply:
As a workaround you can try while being in `mesh_doctor` directory conatining the `.toml`

```
python3 -m pip install --upgrade build
python3 -m build
```

which should produce de `dist/` directory with `.whl` and `.tar.gz`. 

The install command on the `.tar.gz` might also work, as

```
python3 -m pip install dist/mesh_doctor-0.0.1.tar.gz --target /path/to/paraview/lib/pythonX.X'